prowler/lib/outputs/outputs.py

```diff
@@ -139,6 +140,12 @@ def report(check_findings, provider):
                             )
                             file_descriptors["json-ocsf"].write(",")
 
+                        if "html" in file_descriptors:
+                            fill_html(
+                                file_descriptors["html"], finding_output, output_options
+                            )
+                            file_descriptors["html"].write("")
```

Comment:
Why is this needed?

Reply:
When the report is written, first of all the html header is created and the method fill_html is called when creating the report.

Reply:
I know but I mean about ` file_descriptors["html"].write("")`

Reply:
Fact, it's unneeded